From bb8bb785d4390f34e5958202ed2f139bd6156a20 Mon Sep 17 00:00:00 2001
From: Shashwat Ganesh <shashwatganesh@protonmail.com>
Date: Mon, 20 Jan 2025 15:17:43 +0530
Subject: [PATCH] latte: disable exclusive serial port access

---
 latte.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/latte.py b/latte.py
index d2e976e..1667c30 100644
--- a/latte.py
+++ b/latte.py
@@ -113,7 +113,7 @@ def process(encoded_msg,ser):
 def main():
 
     ser = serial.Serial(
-        "/dev/serial/by-id/usb-ZEPHYR_Team_RUDRA_Tarzan_3339511100350023-if00", baudrate=9600, timeout=1)
+        "/dev/serial/by-id/usb-ZEPHYR_Team_RUDRA_Tarzan_3339511100350023-if00", baudrate=9600, timeout=1, exclusive=False)
     while True:
         try:
             raw_data = ser.read(110)
-- 
2.34.1

